assets/src/dashboard/components/storyAnimation/provider.js

```diff
@@ -121,15 +121,15 @@ function Provider({ animations, children, onWAAPIFinish }) {
         // trigger the animation. A workaround to avoid
         // this is to first `cancel` the animation
         // before playing.
-        animation?.cancel();
+        // animation?.cancel();
 
         animation?.play();
       });
     const pause = () =>
       WAAPIAnimations.forEach((animation) => animation?.pause());
     const setCurrentTime = (time) =>
       WAAPIAnimations.forEach((animation) => {
-        const { duration, delay } = animation.effect.timing;
+        const { duration, delay } = animation.effect.getTiming() ?? {};
```

Comment:
`.timing` wasn't reliably returning the timings in the editor for some reason. using `.getTiming()` instead reliably fetched these values